### ISSUE TITLE
WebGPU: GTAO is incompatible with logarithmicDepthBuffer

### ISSUE THREAD

**Issue by ligaofeng0901** [mrdoob/three.js#29797]
### Description

 https://threejs.org/examples/?q=ao#webgpu_postprocessing_ao
In this example, if `logarithmicDepthBuffer` is `true,` it's broken. The error message is below:
<img width="600" alt="image" src="https://github.com/user-attachments/assets/273850d4-4831-4e0a-9466-f3abd7ea0540">
```
Depth stencil state is not present when fragment stage ([ShaderModule "fragment"], [EntryPoint "main"]) is writing to frag_depth.
 - While validating fragment state.
 - While calling [Device].CreateRenderPipeline([RenderPipelineDescriptor "renderPipeline_GTAO_20"]).
```
### Reproduction steps

1.
2.
3.

### Code

```js
renderer = new THREE.WebGPURenderer({
	logarithmicDepthBuffer: true,
});
```

### Live example

* [jsfiddle-latest-release WebGLRenderer](https://jsfiddle.net/3mrkqyea/)
* [jsfiddle-dev WebGLRenderer](https://jsfiddle.net/gcqx26jv/)
* [jsfiddle-latest-release WebGPURenderer](https://jsfiddle.net/8L2jkmx7/)
* [jsfiddle-dev WebGPURenderer](https://jsfiddle.net/L3n1w4yh/)

### Version

0.170.0

### Device

Desktop

**Comment by ligaofeng0901** [comment]
I've found the relevant code of this issue
https://github.com/mrdoob/three.js/blob/dev/examples/jsm/tsl/display/GTAONode.js#L37
If the depth buffer of any render target is disabled, the error occurs, even if in post processing.

**Comment by Mugen87** [comment]
#29799 should fix the warning, however, the Ao does not look right yet. Probably because the depth values differ compared to the normal depth buffer.

**Comment by PoseidonEnergy** [comment]
@Mugen87 I think it may be fixed now, see PR #29870
